src/QueryBuilder/QueryBuilder/helpers/valueBuilder.js

```diff
@@ -26,17 +26,17 @@ export const getFormattedUUID = (value, isInRelatedOperator) => {
     : getQuotedStr(value);
 };
 
-const formatDateToPreview = (dateString, intl) => {
-  const formatedDate = moment(dateString);
+const formatDateToPreview = (dateString, intl, timezone) => {
+  const formattedDate = dayjs.utc(dateString);
 
-  if (formatedDate.isValid()) {
-    return intl.formatDate(formatedDate.toDate(), { dateStyle: 'short' });
+  if (formattedDate.isValid()) {
+    return intl.formatDate(formattedDate.toDate(), { day: '2-digit', month: '2-digit', year: 'numeric', timeZone: timezone });
```

Comment:
we want it to match the QB datepicker style, e.g. `01/01/2020`, rather than the "nicer" options provided by react-intl (`1/1/20`, `Jan 1, 2020`, etc)

Reply:
I did the same thing, Feels Good